docs/design/theory-and-background.md

```diff
@@ -1,23 +1,30 @@
 
 ## About me
 
-I (Tord / SunyataZero) have a Buddhist background, having practiced Buddhism and meditation for over ten years, and being influenced by the [Plum Village tradition](https://plumvillage.org/) which puts emphasis on mindfulness.
+Tord (SunyataZero on github) have a Buddhist background, having practiced Buddhism and meditation for over ten years, and being influenced by the [Plum Village tradition](https://plumvillage.org/) which puts emphasis on mindfulness.
 
 
 ## Background
 
-I, (Tord / SunyataZero) started this project after having worked on a pomodoro application, but then realized that I wanted to focus on mindfulness more than productivity.
-
+Tord started this project after having worked on a pomodoro application, but then realized that I wanted to focus on mindfulness more than productivity.
 
 ## Theory
 
 The application is influenced by Buddhist mindfulness, but the application itself is "secular" and is for everyone to use.
 
+### Buddhist concerns
+
 If you are planning to make changes to the features of this application
 please read the section "Buddhist Concerns" in [this Wikipedia article](https://en.wikipedia.org/wiki/Mindfulness_and_technology).
 
+### Right mindfulness
+
 We need to be aware that there is right mindfulness and wrong mindfulness, right concentration and wrong concentration. Wrong concentration may be that we are so caught-up in our work at the computer.
```

Comment:
so caught-up in our work at the computer that we forget to take care of our body and that we unknowingly build up tension and stress (or something like that...)